keyboards/planck/keymaps/rootiest/config.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
+
+/*
+ * USB/POWER OPTIONS
+ */
+#define USB_MAX_POWER_CONSUMPTION 500
```

Comment:
This line doesn't need to be defined, as it's the default value. 
```suggestion
// #define USB_MAX_POWER_CONSUMPTION 500
```